src/components/GridKeyboardNavigationContext/GridKeyboardNavigationContextHelper.js

```diff
@@ -0,0 +1,51 @@
+import { NAV_DIRECTIONS } from "../../hooks/useFullKeyboardListeners";
+
+export const getDirectionMaps = positions => {
+  const directionMaps = {
+    [NAV_DIRECTIONS.RIGHT]: new Map(),
+    [NAV_DIRECTIONS.LEFT]: new Map(),
+    [NAV_DIRECTIONS.UP]: new Map(),
+    [NAV_DIRECTIONS.DOWN]: new Map()
```

Comment:
why not simple {} ?

Reply:
since I use Objects (refs) as the keys. If I'll use a regular object as a map, the actual value which will be used as key is the result of `.toString()`:
![image](https://user-images.githubusercontent.com/96776835/150313822-8b86de1b-7d12-4555-9365-295234ad1122.png)

on the contrary, [a `Map` can handle objects as keys](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#objects_vs._maps)